backend/src/triage/triage.controller.ts

```diff
@@ -0,0 +1,109 @@
+import { Body, Controller, Get, Param, Patch, Post, Req, UseGuards, Query } from '@nestjs/common';
+import { TriageService } from './triage.service';
+import { SubmitAlertDto } from './dto/submit-alert.dto';
+import { UpdateAlertDto } from './dto/update-alert.dto';
+import { ConvertAlertToCase } from './dto/convert-alert-to-case.dto';
+import { CloseAlertDto } from './dto/close-alert.dto';
+import { AuthGuard } from '@nestjs/passport';
+import { Roles } from '../auth/roles.decorator';
+import { RolesGuard } from '../auth/roles.guard';
+
+@Controller('api/v1/triage/alerts')
+@UseGuards(AuthGuard('jwt'), RolesGuard)
```

Comment:
Utilizing the updated implementation here.

```suggestion
@UseGuards(TazamaAuthGuard)
```